beets/importer.py

```diff
@@ -547,9 +547,22 @@ def _emit_imported(self, lib):
         plugins.send('album_imported', lib=lib, album=self.album)
 
     def emit_created(self, session):
-        """Send the `import_task_created` event for this task.
+        """Send the `import_task_created` event for this task and return a list
+        of tasks which might be returned by plugins.
         """
-        plugins.send('import_task_created', session=session, task=self)
+        tasks = plugins.send('import_task_created', session=session, task=self)
+        if not tasks:
+            tasks = [self]
+        else:
+            # The plugins gave us a list of lists of task. Flatten it.
+            flat_tasks = []
+            for inner in tasks:
+                if isinstance(inner, list):
```

Comment:
Doing so will lead to a crash of beets if a plugin returns something different than a list. This error might be hard to find since the error message could be something totally unexpected. But, on the other hand, a plugin developer should be able to find out what's going wrong. It's the same if the plugin doesn't return an import task, while the error message might be a bit more constructive here.